apps/sms/views/inbox/js/inbox.js

```diff
@@ -118,8 +118,21 @@ var InboxView = {
     Drafts.on('saved', this.onDraftSaved.bind(this));
 
     InterInstanceEventDispatcher.on(
-      'drafts-changed',
-      this.renderDrafts.bind(this, true /* force update */)
```

Comment:
__question:__ mm how about ```init``` method in ```Drafts``` instead that will allow us to keep ```InterInstanceEventDispatcher``` dependency only inside ```Drafts``` and other pieces won't have to do this as well (if/when they need it)?
```
init() {
  InterInstanceEventDispatcher.on('drafts-saved', (draft) => {
    this.request(true).then(() => this.emit('saved', draft));
  });

  InterInstanceEventDispatcher.on('drafts-deleted', (draft) => {
    this.request(true).then(() => this.emit('deleted', draft));
  });
}
```